Extension/package.json

```diff
@@ -1826,7 +1826,10 @@
       "platforms": [
         "win32"
       ],
-      "binaries": []
+      "binaries": [
+        "./bin/cpptools",
```

Comment:
it seems like these should include the extension, i.e. cpptools.exe, since the entry is platform-specific. Was there a reason not to?